test/test_ops.py

```diff
@@ -97,7 +97,7 @@ def test_where(self):
       lambda x: torch.where(x > 0.5, 4, 2),
       lambda x: (x > 0.5).where(4, 2), forward_only=True)
 
-    for shps in [[(10,),(1,),(1,)], [(10,10),(10,),(10,)], [(100,)]*3, [(10,10)]*3]:
+    for shps in [[(8,),(1,),(1,)], [(10,10),(10,),(10,)], [(100,)]*3, [(10,10)]*3]:
```

Comment:
What's the reason behind this shape change?

Reply:
To ensure that there's a float4 test case which covers the linearizer.py:455 change.